packages/core/lib/settings/config.dart

```diff
@@ -47,13 +47,19 @@ flutter_gen:
     enabled: true
     package_parameter_enabled: false
     style: dot-delimiter
+    outputs:
+      class_name: Assets
```

Comment:
Becoz you checked in this line [ in Asset](https://github.com/FlutterGen/flutter_gen/pull/280/files#diff-8dc8f78c31449fdef325c4ed0e8da50067716f50c6370fc9e477e3627219e63fR332), so the default config no need `outputs.class_name` param, right?